backend/api/projects.py

```diff
@@ -406,3 +405,43 @@ def delete_comment(project_id, comment_id):
         raise error
     except Exception as err:
         raise ApiException(str(err), 500)
+
+
+@projects.route('/api/projects/<uuid:project_id>/share/<user_mail>', methods=['POST'])
+@login_required
+def share_via_email(user_mail):
+    """Creates and saves notification object
+
+        Returns:
+            res: Notification object as json
+    """
+    user = g.user_datastore.get_user(user_mail)
+    if not user:
+        return make_response("Unknown User with Email-address: " + user_mail, 404)
+    res = g.projects.find_one({'_id': project_id})
+    if not res:
+        raise ApiException("Project not found", 404)
+
+    description = res["title"] + "was shared with you"
+    link = "/projects/" + str(project_id)
```

Comment:
is this the way the link looks at the frontend? I heard something different today